Arduino/Esp32/Main/Main.ino

```diff
@@ -685,7 +685,7 @@ void pedalUpdateTask( void * pvParameters )
         }
         if(dap_config_st.payLoadPedalConfig_.Simulate_ABS_trigger==1)
         {
-          if(joystickNormalizedToInt32>8000)
+          if(joystickNormalizedToInt32>dap_config_st.payLoadPedalConfig_.Simulate_ABS_value*100)
```

Comment:
An uint8 value is multiplied by 100. Thats potentially dangerous as it could lead to overflow. Could you please add an int32 cast here, e.g.:

((int32)dap_config_st.payLoadPedalConfig_.Simulate_ABS_value) * 100

Reply:
sure! thanks!

Reply:
> An uint8 value is multiplied by 100. Thats potentially dangerous as it could lead to overflow. Could you please add an int32 cast here, e.g.:
> 
> ((int32)dap_config_st.payLoadPedalConfig_.Simulate_ABS_value) * 100

done and also upload the canvas control plugin
![image](https://github.com/ChrGri/DIY-Sim-Racing-FFB-Pedal/assets/48719709/7e9a2842-25c4-4497-a7cf-6953657ee1f5)

Reply:
Awesome. Since a field to the config structure was added, please increment the revision DAP_VERSION_CONFIG to 109 in the arduino and plugin code

Arduino code:
https://github.com/ChrGri/DIY-Sim-Racing-FFB-Pedal/blob/2263fd210005ca66426be1766a837a62ac9c646a/Arduino/Esp32/Main/DiyActivePedal_types.h#L6

Plugin code:
https://github.com/ChrGri/DIY-Sim-Racing-FFB-Pedal/blob/2263fd210005ca66426be1766a837a62ac9c646a/SimHubPlugin/DataPluginDemo.cs#L470

https://github.com/ChrGri/DIY-Sim-Racing-FFB-Pedal/blob/2263fd210005ca66426be1766a837a62ac9c646a/SimHubPlugin/SettingsControlDemo.xaml.cs#L37

Reply:
all done! many thanks!

Reply:
With the more interactive canvas control, the vertical sliders can be removed and the rectangle positions have to be updated when the config is received from the pedal:
![image](https://github.com/ChrGri/DIY-Sim-Racing-FFB-Pedal/assets/21274895/c13f8547-da29-425c-b86e-ecf0eeb100f4)

You would do me a favour by implementing the aforementioned changes.

Reply:
> With the more interactive canvas control, the vertical sliders can be removed and the rectangle positions have to be updated when the config is received from the pedal: ![image](https://private-user-images.githubusercontent.com/21274895/292658130-c13f8547-da29-425c-b86e-ecf0eeb100f4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDM0MDEwNjQsIm5iZiI6MTcwMzQwMDc2NCwicGF0aCI6Ii8yMTI3NDg5NS8yOTI2NTgxMzAtYzEzZjg1NDctZGEyOS00MjVjLWI4NmUtZWNmMGVlYjEwMGY0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjI0VDA2NTI0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIwNDFkYmRmM2ExZWM4ZGIxNGMxZmZiZDJkZGUyMjlmNzRhZDA3NGY1MzViN2Q2ZjI0NGM4MWY3MjlkZDY3MTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.j89o33GbD4q-dSolXiiL8k2wlNODQZ64UCw0KgcDUAg)
> 
> You would do me a favour by implementing the aforementioned changes.

I think the config from pedal will auto update.
will hide those slide first, when I have time, will clean the code then.
![canvas_control_reading_from_config](https://github.com/ChrGri/DIY-Sim-Racing-FFB-Pedal/assets/48719709/412893a8-187f-4f97-bbb2-793e064898ea)

Reply:
the force vs travel slider code clean up already done